calculate_average.js

```diff
@@ -0,0 +1,14 @@
+
+const calculateAverage = function calculateAverage(grades) {
+
+  let total = 0;
+  for (let i = 0; i < grades.length; i++) {
+    total += grades[i];
+  }
+
+  let avg = total / grades.length;
+
+  return avg;
+};
+
+export default calculateAverage;
```

Comment:
Can we add a new line at the end of the file, as per our team's style guidelines?